c_src/workitems.cc

```diff
@@ -144,9 +144,10 @@ OpenTask::OpenTask(
     ERL_NIF_TERM& _caller_ref,
     const std::string& db_name_,
     rocksdb::Options *Options_,
+    bool use_ttl_,
```

Comment:
Use 4 spaces instead of 1 tab for the indent.

Reply:
Are you sure it's a tab there ? Because for me shows 4 spaces

Reply:
Sorry I pointed out the wrong file.
please check "bool use_ttl;" in c_src/workitems.h
here a tab is used as an indent.